k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -343,7 +343,15 @@ public void onLoaderReset(Loader<List<Recipient>> loader) {
     }
 
     public boolean hasUncompletedText() {
-        return !TextUtils.isEmpty(currentCompletionText());
+        String currentCompletionText = currentCompletionText();
+        if(TextUtils.isEmpty(currentCompletionText)) {
+            return false;
+        }
+        // TODO this is sort of a hack, but it's reliable and there is no way to access this information directly atm
+        // noinspection UnnecessaryLocalVariable
+        boolean isPlaceholderString
```

Comment:
Usually, I'm a fan of local variables to give a name to things. But this really wants to be a method instead.
That way you don't have to ignore the warning and the comment is less likely to end up in the wrong place when someone modifies the method at a later point.

Generally, ignoring warnings is a sign of code smell. Sometimes there are legitimate reasons to do it, but this is not one of them.

Also, code style issues.
* Space missing after `if`.
* When you need to add line breaks the "operator" should be at the end of the line.